### ISSUE TITLE
Can't click to sort after adding <span> to label

### ISSUE THREAD

**Issue by alantliang** [aslagle/reactive-table#361]
Hi, thanks a lot for creating this package, it's awesome! One quick question:

I am trying to use a template for the label:
```
<template name="foo">
	<span style="font-size: 10px">
	Foooooooooooooooooooooooooo
	</span>
</template>
```

And my label is set correctly: 
```
        ...
        label: Template.foo
        ...
```

Everything works as intended except that I can no longer click on "Foooooooooooooooooooooooooo" to sort, I can only click on the outer div. Am I missing something obvious? Thanks!

**Comment by aslagle** [comment]
You're not missing anything, this is a bug. I'm surprised no one came across this before. Thanks for reporting it, I think it will be easy to fix.

**Comment by aslagle** [comment]
This is fixed in version 0.8.28.

**Comment by alantliang** [comment]
great thanks!
